lib/medical_records/bb_internal/client.rb

```diff
@@ -136,6 +136,18 @@ def get_bbmi_notification_setting
       response.body
     end
 
+    # Retrieves the patient information by user ID.
+    # @return [Hash] A hash containing the patient's details
+    #
+    def get_patient
+      response = perform(:get, "usermgmt/patient/uid/#{@session.user_id}", nil, token_headers)
+      patient = response.body
+
+      raise Common::Exceptions::ServiceError.new(detail: 'Patient not found') if patient.blank?
```

Comment:
If the patient does not exist, will the response body be blank? Or will it be populated with some kind of error?

Reply:
The body is indeed just blank if the patient does not exist. Comes back with a 204 and no body